src/Reflex/Profiled.hs

```diff
@@ -144,16 +145,32 @@ instance Reflex t => Reflex (ProfiledTimeline t) where
   unsafeBuildIncremental (ProfiledM a0) (Event_Profiled a') = coerce $ unsafeBuildIncremental a0 a'
   mergeIncremental = Event_Profiled . mergeIncremental . (unsafeCoerce :: Incremental (ProfiledTimeline t) (PatchDMap k (Event (ProfiledTimeline t))) -> Incremental t (PatchDMap k (Event t)))
   mergeIncrementalWithMove = Event_Profiled . mergeIncrementalWithMove . (unsafeCoerce :: Incremental (ProfiledTimeline t) (PatchDMapWithMove k (Event (ProfiledTimeline t))) -> Incremental t (PatchDMapWithMove k (Event t)))
```

Comment:
I think these can be derived with:
```haskell
case (incrementalCoercion, eventCoercion) of
  (Coercion, Coercion) -> Coercion
```
I'd certainly hope the newtype coercion axioms are defined for data families too!